apps/doji-frontend/src/modules/payment/select/index.tsx

```diff
@@ -122,6 +121,12 @@ const SelectPaymentPanel = () => {
           }}
         />
       ))}
+
+      {data.length == 0 && (
+        <Link href="/balance/new">
+          <Tables content="Add a new card" avatar={{ children: <FaPlus /> }} />
```

Comment:
ไม่แน่ใจว่าการมี การเพิ่ม card ได้ 2 ที่มันจะทับซ้อนกันไหม น่าจะต้องเอาอันใดอันหนึ่งออกหรือป่าว (ส่วนตัวคิดว่าควรเอาอันเล็กข้างบนออก)

Reply:
Agree, but I just follow the order of @saenyakorn